packages/clay-tabs/src/Item.tsx

```diff
@@ -0,0 +1,92 @@
+/**
+ * © 2019 Liferay, Inc. <https://liferay.com>
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+import classNames from 'classnames';
+import React from 'react';
+
+export type ElementType = 'a' | 'button';
+
+interface IProps extends Omit<React.HTMLAttributes<HTMLLIElement>, 'onClick'> {
+	/**
+	 * Flag to indicate if the component is active or not.
+	 */
+	active?: boolean;
+
+	/**
+	 * Tab Item component to render. Can be an 'anchor' or a 'button'.
+	 */
+	component?: ElementType;
```

Comment:
I intend to remove this API to use the new `LinkOrButton` component being introduced by #2225, so the component tag will be decided when passing `href` or `onClick`.